ioctl.c

```diff
@@ -813,6 +841,7 @@ cryptodev_ioctl(struct file *filp, unsigned int cmd, unsigned long arg_)
 	struct crypt_priv *pcr = filp->private_data;
 	struct fcrypt *fcr;
 	struct session_info_op siop;
+	struct cphash_op cphop;
```

Comment:
should this be ifdef guarded?

Reply:
Yes it should.  I'll fix it up right away.

Reply:
About the kernel version, I think we should keep it for the time being.  DD-WRT, for example, still uses old linux kernels.  Unless there's something else that already needs > 2.6.29.  I can't test the ancient versions myself, though.